seismic_zfp/utils.py

```diff
@@ -77,7 +77,7 @@ def read_range_blob(file, offset, length):
     return file.download_blob(offset=offset, length=length).readall()
 
 
-def generate_fake_seismic(n_ilines, n_xlines, n_samples, min_iline=0, min_xline=0):
+def generate_fake_seismic(n_ilines, n_xlines, n_samples, min_iline=0, min_xline=0, min_sample=0):
```

Comment:
What is min_sample here?

Reply:
it's the minimum sample value for the z dimension of the cube. min_sample is the (missing) equivalent to the two other dimension of the cube, e.g. min_iline and min_xline